pkg/apis/core/types_cloudprofile.go

```diff
@@ -187,15 +187,15 @@ const (
 	VolumeClassPremium string = "premium"
 )
 
-// VersionClassification is the logical state of a version according to https://github.com/gardener/gardener/blob/master/docs/operations/versioning.md
+// VersionClassification is the logical state of a version.
 type VersionClassification string
 
 const (
 	// ClassificationPreview indicates that a version has recently been added and not promoted to "Supported" yet.
 	// ClassificationPreview versions will not be considered for automatic Kubernetes and Machine Image patch version updates.
 	ClassificationPreview VersionClassification = "preview"
 	// ClassificationSupported indicates that a patch version is the recommended version for a shoot.
-	// Using VersionMaintenance (see: https://github.com/gardener/gardener/docs/operation/versioning.md) there is one supported version per maintained minor version.
+	// Using VersionMaintenance there is one supported version per maintained minor version.
```

Comment:
THB, I don't know what is this `VersionMaintenance` - is it a func; or?

That's why in my previous comment I suggested the following:

```suggestion
	// Only one "supported" version is allowed per minor version.
```